test-res/org/pentaho/metadata/sampledata.properties

```diff
@@ -0,0 +1,4 @@
+#HSQL Database Engine 2.3.2
```

Comment:
It is not mistake.  Please see
https://github.com/pentaho/pentaho-metadata/blob/master/ivy.xml#L79

You are completely right, that the changes can be ignored.  The file was added to .gitignore.